### ISSUE TITLE
Diagram could not show.

### ISSUE THREAD

**Issue by Clubiy** [QSD-Group/QSDsan#128]
Please replace prompts with your situation, add a note (e.g., NA) if not applicable

**Describe the problem**
When I followed the tutorial to complete all the environment configuration step by step, I found that I could not draw the system structure diagram, like the screenshot below. Is it Graphviz's problem?

**To Reproduce**
Steps to reproduce the behavior, e.g.:
I tried to change the version of python and qsdsan, python3.11 and python3.9, qsdsan1.3.1 and qsdsan1.4.0, but all faild.

**Screenshots**
![image](https://github.com/user-attachments/assets/e67bc68d-f06b-4179-96f7-019bb6a1e3e9)
**Environment (please complete the following information):**
1. Versions of the following packages:
     - QSDsan=1.3.1 or 1.4.1
     - EXPOsan (if applicable)=1.4.1
     - BioSTEAM=2.49.4
     - Thermosteam=0.49.2
     - Graphviz=2.50.0

    If you are using pip-installed ones, you can get the info by `pip show <PACKAGE_NAME>` (e.g., `pip show qsdsan`).

    If you have cloned any of the following packages, please make sure your local and remote are in sync (either pull remote changes to local or push local changes to remote), then include links or point to the branches/forks you are using.

2. Operating systems and version [e.g. macOS 12.1, Windows 11]

**Additional context**

**Comment by Clubiy** [comment]
![image](https://github.com/user-attachments/assets/0af05dea-dbdd-4e8e-88f9-92dc1f5380ab)

**Comment by yalinli2** [comment]
yes it's most likely a graphviz issue, are you using Windows or Mac?

also, if you set `biosteam.preferences.raise_exception` ti `True`, then run the codes to generate the graph, did you get more detailed exceptions?

**Comment by Clubiy** [comment]
Using Mac and Windows and Linux, all the same problem. And I set this parameter, but no exception was raised. Right now, I have no idea to solve this problem, so I decide to find my old machine, the environment there maybe working. Thank you for your support, and if you find a solution later, please let me know in time. It's a great help. Thank you!

来自CLubiy的iPhone
hope you have a good time!
________________________________
发件人: Yalin ***@***.***>
发送时间: Thursday, December 26, 2024 8:56:28 PM
收件人: QSD-Group/QSDsan ***@***.***>
抄送: Clubiy ***@***.***>; Author ***@***.***>
主题: Re: [QSD-Group/QSDsan] #BUG# Diagram could not show. (Issue #128)


yes it's most likely a graphviz issue, are you using Windows or Mac?

also, if you set biosteam.preferences.raise_exception ti True, then run the codes to generate the graph, did you get more detailed exceptions?

―

**Comment by yalinli2** [comment]
OK! I'm closing it for now - feel free to reopen/open a new one if it comes back. Thanks for letting me know!
